WcaOnRails/app/views/competitions/_nearby_competitions.html.erb

```diff
@@ -11,6 +11,9 @@
   <div class="col-sm-9">
     <% if @competition.has_date? && @competition.has_location? %>
       <% nearby_competitions = @competition.nearby_competitions(Competition::NEARBY_DAYS_WARNING, Competition::NEARBY_DISTANCE_KM_WARNING)[0, 10] %>
+      <% if !current_user.can_view_hidden_competitions? %>
```

Comment:
Before @jonatanklosko says it... maybe use an `unless` here?

You can also have your `<% ... %>` span multiple lines:

```ruby
<%
  unless current_user.can_view_hidden_competitions?
    nearby_competitions = nearby_competitions.select(&:isConfirmed?)
  end
%>
```

However, ruby also has a one line if statement that might be nice here:

```ruby
<% nearby_competitions = nearby_competitions.select(&:isConfirmed?) unless current_user.can_view_hidden_competitions? %>
```

As a high level thought... I'm not wild about enforcing this access at the view level... it would be nice if instead, it were enforced in the `nearby_competitions` method, but that's a model method, and it does not have access to the currently logged in user.